contrib/kind.sh

```diff
@@ -110,6 +110,7 @@ usage() {
     echo "                 [-cn | --cluster-name |"
     echo "                 [-ehp|--egress-ip-healthcheck-port <num>]"
     echo "                 [-is | --ipsec]"
+    echo "                 [-cm | --compact-mode]"
```

Comment:
++ nit: also will be good to update `kind.md` doc file (docs go stale if not kept upto date as we add new options).